frontend/src/ui/Blocks/Series.tsx

```diff
@@ -356,6 +696,9 @@ const GridTile: React.FC<GridTypeProps> = ({ event, basePath, active }) => {
                 {/* `new Date` is well defined for our ISO Date strings */}
                 <RelativeDate date={new Date(date)} isLive={event.isLive} />
             </div>
+            {isList && event.description && <p css={ellipsisOverflowCss(3)}>
+                {event.description}
+            </p>}
```

Comment:
The description should be smaller font and should also say "no description" if there is none. Luckily you can use `SmallDescription` from `src/ui/metadata.tsx`.